draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -8373,7 +8379,7 @@ Content-Range: exampleunit 11.2-14.3/25
    though an origin server &MAY; generate content of zero length.
    If no content is desired, an origin server ought to send
    <x:dfn>204 (No Content)</x:dfn> instead.
-   For CONNECT, no content is allowed because the successful result is a
+   For CONNECT, there is no content because the successful result is a
```

Comment:
"no content is allowed" can parse as "the situation where there is no content is a permitted situation" or as "it is forbidden/impossible to have content".  I propose this rewording because this is not an arbitrary restriction but rather an inherent property of how connect works.